appserver/appclient/client/acc/src/main/java/org/glassfish/appclient/client/CLIBootstrap.java

```diff
@@ -535,6 +541,27 @@ StringBuilder format(final StringBuilder commandLine, final boolean useQuotes, f
         }
     }
 
+    private class CommandLineArgument extends CommandLineElement {
```

Comment:
```suggestion
    class CommandLineArgument extends CommandLineElement {
```
I'd rather see this as package-accessible and have reflection dropped in UTs.